src/components/Combobox/Combobox.tsx

```diff
@@ -243,6 +250,28 @@ function Combobox<Item>({
               isOpen={isOpen && results.length > 0}
               {...getMenuProps()}
             >
+              {withClearSelectionControl && (
+                <Box
+                  as="li"
```

Comment:
In the video, i noticed that this button gets hidden when the combobox is empty. I cannot see that in the code.
Also, when scrolling this button gets hidden behind the list items.

Reply:
Forgot to push it. Please re-review